package-lock.json

```diff
@@ -1,8 +1,19 @@
 {
   "name": "osrm",
-  "version": "5.13.0-latest.1",
+  "version": "5.14.0-latest.1",
```

Comment:
We should not include the `package-lock.json` diff. Seems like this was accidentally committed?